### PR TITLE
Add Left & Right panel

### PR DESCRIPTION
**Pull request by danielduhh**
### Description:

This adds data to both the left and right panel, which required a bunch of small changes to the data. The right panel should work once duplicate id's are fixed re: #17

Left panel "must see" events are categorized in order of importance.

I also fixed the failed tests reported in #16 

### UI images:

Screen Shot 2019-07-15 at 4 46 58 PM

Screen Shot 2019-07-15 at 4 59 10 PM

### Unit Test Approach:

Describe unit tests being added with that PR.

### Test Results:
- [x] Did you run `npm test` and did the results pass?
- [ ] Did you add unit tests that cover your changes?

Describe other (non-unit) test results here.

### CODE REVIEW

**Review by stephencerqueira — APPROVED** [review]
LGTM

File: src/components/LeftDrawer/LeftDrawer.js [hunk 1]
```diff
@@ -9,29 +9,30 @@ import Divider from '@material-ui/core/Divider';
 import ListItem from '@material-ui/core/ListItem';
 import ListItemText from '@material-ui/core/ListItemText';
 import {
-  toggleLeftDrawer, toggleRightDrawer, setTabValue, selectLeftMenuItem,
```
Comment (stephencerqueira) [inline src/components/LeftDrawer/LeftDrawer.js]: There's no need to toggle right drawer or set the tab value here anymore?

Comment (stephencerqueira) [inline src/components/LeftDrawer/LeftDrawer.js:12]: No need to toggle the right drawer or set the tab value here anymore?

**Review by stephencerqueira — COMMENTED** [review]
One comment about LeftDrawer callouts. Otherwise, LGTM

**Review by cbessee — APPROVED** [review]
Looks good!

File: src/components/LeftDrawer/LeftDrawer.js (hunk 1 above)
Comment (danielduhh) [reply src/components/LeftDrawer/LeftDrawer.js:12]: We used to have a custom value in the list that would open the right drawer; I got rid of that for now and just added the list of "must see" features. If we want to add the "Event" list icon to open the right menu later on we can

**Pull request merged**